src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,44 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGTH = 6;
```

Comment:
The constant for minimum login length is named MIN_LENGTH, which could be confusing as it is used for both login and password. Consider renaming it to something more specific, such as MIN_LOGIN_LENGTH and MIN_PASSWORD_LENGTH, to improve code readability and avoid potential errors in the future.